docs/developer-docs/functionality/http_requests/http_requests-how-it-works.md

```diff
@@ -1,71 +1,75 @@
-# Technology Overview -- How Canister HTTP Requests Work
+# Technology Overview — How Canister HTTP Outcalls Work
```

Comment:
```suggestion
# Technology Overview — How Canister HTTPS Outcalls Work
```
If you search for 'HTTP ' (including space) you should find all instances of HTTP without an S